src/VettedGate.sol

```diff
@@ -300,6 +304,11 @@ contract VettedGate is
         return MerkleProof.verifyCalldata(proof, treeRoot, hashLeaf(member));
     }
 
+    /// @inheritdoc IVettedGate
+    function getInitializedVersion() external view returns (uint64) {
```

Comment:
do we need this for 1st version of the contract?

Reply:
Yes. I will later add them to the other contracts behind proxy